pkg/agent/metrics/prometheus.go

```diff
@@ -67,6 +67,24 @@ var (
 		Help:           "Flow count for each OVS flow table. The TableID is used as a label.",
 		StabilityLevel: metrics.STABLE,
 	}, []string{"table_id"})
+
+	OVSFlowOpsErrorCount = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "antrea_agent_ovs_flow_ops_error_count",
+			Help:           "Number of OVS flow operation errors.",
+			StabilityLevel: metrics.STABLE,
+		},
+		[]string{"operation"},
+	)
+
+	OVSFlowOpsDuration = metrics.NewSummaryVec(
```

Comment:
Not your change. There are other summary/summary vector metrics in Antrea. As per [Kubernetes metric overhaul](https://github.com/kubernetes/enhancements/issues/1206), it is recommended to use histograms instead of summaries.  And the summary metrics are tagged for deprecation. 
Main advantages with histogram are [aggregation and inexpensive](https://prometheus.io/docs/practices/histograms/#:~:text=The%20essential%20difference%20between%20summaries,the%20server%20side%20using%20the). Any comments @ksamoray ?

Reply:
@antoninbas @tnqn  any thoughts on above?

Reply:
I am not an expert, but we have been using the `STABLE` stability level for all these metrics. According to that [contract](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes), *the type of the metric will not be modified*. So while I am fine with using histograms instead of summaries for *new* metrics,  do we actually want to update existing metrics to use histograms? Or should we follow the guidelines and deprecate the old metrics while introducing new ones with the histograms type?

Reply:
As for new metrics in this PR, I'll follow the recommendation to use histgram.

Reply:
So while I am fine with using histograms instead of summaries for _new_ metrics, do we actually want to update existing metrics to use histograms? Or should we follow the guidelines and deprecate the old metrics while introducing new ones with the histograms type?

Thanks for the response. Yes, making the current metric deprecated and add the new metrics with histogram type is suggested. After a couple of releases, removing deprecated metrics is suggested as a guideline. I am wondering this makes sense if there are consumers (w/dashboards) or third party software dependent on these metrics. If not, could we just update them with histogram type?

Reply:
Given that we probably have few users (or none) relying on these metrics. I don't think I would be opposed to just switching the metric type, providing we sent the appropriate notices on Slack and the mailing list and waited a couple of days for feedback. In the future, it may be a good idea to not tag these metrics as `STABLE` right away...

Reply:
Not defining them as STABLE sounds good. We should define new metrics as ALPHA and move them to STABLE after a release or two; or when confident that these were being used effectively.   
Let me open an issue and post this in slack and mailing list.

Reply:
Agreed to go with Alpha in a few first releases.
I'll make a change to this PR.
@srikartati @antoninbas  But do you think we also need to make existing metrics Alpha?

Reply:
I think it is better to make them alpha and turn them into STABLE after the next release.

Reply:
Thank you for your comments.